argus/resources/windows/schedule_installer.bat

```diff
@@ -1,4 +1,4 @@
-schtasks /CREATE /TN "cloudbaseinit-installer" /SC ONCE /SD 01/01/2020 /ST 00:00:00 /RL HIGHEST /RU CiAdmin /RP Passw0rd /TR "powershell C:\\installcbinit.ps1  -serviceType %1 -installer %2" /F
+schtasks /CREATE /TN "cloudbaseinit-installer" /SC ONCE /SD 01/01/2020 /ST 00:00:00 /RL HIGHEST /RU CiAdmin /RP Passw0rd /TR "powershell C:\\installcbinit.ps1 -installer %2" /F

```

Comment:
This script is called as follows: `schedule_installer.bat -installer CloudbaseInitSetup_*_*.msi`. So, `%1` refers to `"-installer"` and `%2` - `"CloudbaseInitSetup_*_*.msi"`.